test/Microsoft.AspNetCore.Mvc.Core.Test/project.json

```diff
@@ -8,6 +8,7 @@
     "dotnet-test-xunit": "2.2.0-*",
     "Microsoft.AspNetCore.Http": "1.1.0-*",
     "Microsoft.AspNetCore.Mvc": "1.1.0-*",
```

Comment:
This will pull in `Mvc.Core` (transitively...FYI, you can check the References node tree for this), so the below addition is not required.